systemtest/src/test/resources/oauth2/prepare_keycloak_operator.sh

```diff
@@ -6,10 +6,64 @@ KEYCLOAK_INSTANCE_NAMESPACE=$3
 
 SCRIPT_PATH=$(dirname "${BASH_SOURCE[0]}")
 
+function create_keycloak_cr() {
```

Comment:
wouldn't be better to have this in yaml?

Reply:
Currently I think it's good as is, because it's workaround due to issue https://github.com/keycloak/keycloak-operator/issues/426
Once this issue is resolved, I think we could start using (again) example-keycloak.yaml which we used before. 
I will make a note there.